### ISSUE TITLE
Too slowly picking local changes

### ISSUE THREAD

**Issue by memeplex** [syncthing/syncthing-android#1164]
I'm trying to replace dropbox to keep my phone an my linux box in sync but I'm experiencing very large update latencies (this is for small text files):

1. If I change a file in my linux box the change is propagated to my phone in about 2-3 seconds, that's great.

2. If I change a file in my phone it takes about 30s to 60s to reach my linux box.

Both sides watching is enabled. I tried turning on every battery-sucking experimental feature in the syncthing app but to no avail.

**Comment by Catfriend1** [comment]
I experienced very fast sync times between my qnap box and android phone. Max. 20 seconds with both devices connected via wifi+lan to the same network. Only one folder waited for the scan time so I suspect some fswatcher issue in the upstream syncthing that probably already got fixed in the 14.49rc in the meantime as the cause. I suggest waiting for syncthing 14.49 to release and then report back here if the issue still persists. Logs or reproducer are appreciated.
